hardware/raspberrypi/booteeprom.md

```diff
@@ -44,17 +48,18 @@ sudo systemctl unmask rpi-eeprom-update
 
 The `FREEZE_VERSION` option in the EEPROM config file may be used to indicate that the EEPROM should not be updated on this board. 
 
+Note that by default, updating the bootloader (automatically or manually) will retain any custom configuration options of the previous installed version. You can override the migration by manually updating and using the `-d` option with `rpi-eeprom`. This will force the updated bootloader to use its inbuilt defaults.
```

Comment:
`rpi-eeprom` -> `rpi-eeprom-update`

`rpi-eeprom` is the name of the package, which contains the eeprom bootloader images and the `rpi-eeprom-update` and `rpi-eeprom-config` commands.

Reply:
yes that should say rpi-eeprom-update